Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official Node.js image from the Docker Hub
+FROM node:18
+
+# Create and set the working directory inside the container
+WORKDIR /usr/src/app
+
+# Copy package.json and package-lock.json to the working directory
+COPY package*.json ./
+
+# Install the Node.js dependencies
+RUN npm install
+
+# Copy the rest of the application code to the working directory
+COPY . .
+
+# Define the command to run the application
+CMD ["node", "src/index.js"]
```

Comment:
Should we build and then run the compiled/built `index.js` instead here?

Reply:
I don't think there is a huge difference for the size of the codebase currently, but we should keep it in mind for future reference.